styles/button/btnhover.css

```diff
@@ -0,0 +1,59 @@
+.button:hover{
+    background-color: white;
+    color: var(--default-color);
+    transform: scale(1.1);
+}
+.button-primary:hover{
+    background-color: var(--secondary-color);
+    color: var(--primary-color);
+} 
+.button-secondary:hover{
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+
```

Comment:
for better CSS don't give any extra space even at the end line of the css file.

Reply:
Sure, thank you! done.